src/sas/sasgui/guiframe/local_perspectives/data_loader/data_loader.py

```diff
@@ -204,7 +204,7 @@ def get_data(self, path, format=None):
                                  info="info")
             except NoKnownLoaderException as e:
                 exception_occurred = True
-                error_message = "Loading data failed!\n" + e.message
+                error_message = "Loading data failed!" + e.message
```

Comment:
Don't you need a space after `failed!`? Or do you know that e.message starts with a space?

Reply:
A NoKnownLoaderException message always starts with a leading carriage return.